src/actions/index.js

```diff
@@ -1,26 +1,38 @@
-import axios from 'axios';
+import { config } from '../config';
+import firebase from 'firebase';
 
 export const FETCH_BEER = 'FETCH_BEER';
 export const UPDATE_BEER = 'UPDATE_BEER';
 
-const url = 'http://localhost:3000/beers'
-
-export function fetchBeers(){
-
-    const request = axios.get(url);
-
-    return{
-        type: FETCH_BEER,
-        payload: request
-    }
+firebase.initializeApp(config);
+
+const db = firebase.firestore();
+db.settings({
+  timestampsInSnapshots: true,
+});
+
+export function fetchBeers() {
+  const data = db
+    .collection('Beers')
+    .get()
+    .then(snapshot =>
+      snapshot.docs.map(doc => ({ id: doc.id, data: doc.data() })),
+    );
+
+  return {
+    type: FETCH_BEER,
+    payload: data,
+  };
 }
```

Comment:
You are using 2 spaces in this file, 4 spaces in some other files. Please configure eslint and prettier and keep indent consistent